ui/pages/bridge/prepare/bridge-cta-button.tsx

```diff
@@ -36,18 +48,40 @@ export const BridgeCTAButton = () => {
 
   const { submitBridgeTransaction } = useSubmitBridgeTransaction();
 
-  const { isNoQuotesAvailable, isInsufficientBalance } =
-    useSelector(getValidationErrors);
+  const {
+    isNoQuotesAvailable,
+    isInsufficientBalance: isInsufficientBalance_,
+    isInsufficientGasBalance: isInsufficientGasBalance_,
+    isInsufficientGasForQuote: isInsufficientGasForQuote_,
+  } = useSelector(getValidationErrors);
```

Comment:
I just noticed that some of these are functions, usually selectors just return data. I think these probably should go in a hook or something. Not a blocker though.